### PR TITLE
Add load test for line diagrams

### PR DESCRIPTION
**Pull request by thecristen**
#### Summary of changes
**Asana Ticket:** [Add docs/process around load testing](https://app.asana.com/0/385363666817452/1200761946703394/f)

Enhances our load testing script by

1. Linking to its docs in a relevant spot (in this case the PR template)
2. Updating the Locust dependency (it was very outdated!)
3. Adding additional tests on the line diagram page. 

Much of that script is adapted from the `locustfile.py` in the [API repo](https://github.com/mbta/api/blob/master/load_tests/locustfile.py). Additional logic is added to simulate the multiple JS-driven API calls that happen simultaneously on the page. There is also logic inspecting the request responses for certain values, as we expect to see not only a valid non-error HTTP response, also data! It's a little convoluted but it does okay ✨

### CODE REVIEW

File: apps/site/load_tests/README.md [hunk 1]
```diff
@@ -4,13 +4,14 @@ This directory contains our load testing scripts which use [Locust](https://locu
...
 
-Follow the [Locust instructions](https://docs.locust.io/en/stable/installation.html) for installing system dependencies (notably `libev` on the macOS).
+Follow the [Locust instructions](https://docs.locust.io/en/2.1.0/installation.html) for installing system dependencies (notably `libev` on the macOS).
 
 Install required libraries:
 
     cd apps/site/load_tests/
     pip3 install -r requirements.txt
 
+If using `asdf`, you may have to run `asdf reshim python 3.8.5 locust` before using Locust in your shell.
```
Comment (arkadyan) [inline apps/site/load_tests/README.md:16]: Nit: You might want a blank line between this line and the following header.

File: apps/site/load_tests/locustfile.py [hunk 2]
```diff
@@ -1,6 +1,5 @@
-from locust import HttpLocust, TaskSet, task
-from page import Page
```
Comment (arkadyan) [inline apps/site/load_tests/locustfile.py:2]: It looks like the `page.py` file is no longer used anywhere and can probably just be deleted.

File: apps/site/load_tests/locustfile.py [hunk 3]
```diff
@@ -46,8 +45,19 @@ def paris(self):
...
+    tasks = {
+        TransitNearMe: 3,
+        LineDiagramCR: 5,
+        LineDiagramBus: 5,
+        LineDiagramSubway: 3,
+        LineDiagramFerry: 1
+    }
+
+    min_wait = 1000  # 1 seconds
+    max_wait = 15000  # 15 seconds
```
Comment (arkadyan) [inline apps/site/load_tests/locustfile.py:63]: Nit: Missing newline at EOF.

File: apps/site/load_tests/locustfile.py [hunk 4]
```diff
@@ -1,6 +1,5 @@
-from locust import HttpLocust, TaskSet, task
-from page import Page
-
+from locust import HttpUser, TaskSet, task
+from linediagram import LineDiagramTaskSet
 
 class TransitNearMe(TaskSet):
```
Comment (arkadyan) [inline apps/site/load_tests/locustfile.py:4]: Not related to this commit, so feel free to ignore, but I could see the `TransitNearMe` class moving to its own file.

**Review by arkadyan — APPROVED** [review]
This is fun! I added a few comments, but they're only nitpicks and tangents, nothing blocking.

**Review by alicia-mbta — APPROVED** [review]
As far as I can see, it looks good to me too!
It was cool to see the charts updating in real time 📈

**Pull request merged**